draft-irtf-cfrg-opaque.md

```diff
@@ -1673,8 +1673,8 @@ parameters that are needed to prevent cross-protocol or downgrade attacks.
 
 Absent an application-specific profile, the following configurations are RECOMMENDED:
 
-- OPRF(ristretto255, SHA-512), HKDF-SHA-512, HMAC-SHA-512, SHA-512, Scrypt(32768,8,1), ristretto255
-- OPRF(P-256, SHA-256), HKDF-SHA-256, HMAC-SHA-256, SHA-256, Scrypt(32768,8,1), P-256
+- OPRF(ristretto255, SHA-512), HKDF-SHA-512, HMAC-SHA-512, SHA-512, Argon2id, ristretto255
```

Comment:
We need to include parameters for Argon2id here, since this is a recommended profile that applications are meant to be able to use without doing anything else.

Reply:
In particular, I think we ought to go with the FIRST RECOMMENDED option in RFC9106. Those options were recommended for a reason, and it makes sense to me that we reuse them here under the same reasoning.